Selenium.Axe/Selenium.Axe/HtmlReport.cs

```diff
@@ -12,149 +12,239 @@ public static class HtmlReport
+                                          expandoText.innerHTML = ""-"";
+                                      }
+                                  })
+                              }
+  
+                              var thumbnail = document.getElementById(""screenshotThumbnail"");
+                              var modal = document.getElementById(""modal"");
+                              var modalimg = modal.getElementsByTagName(""img"")[0]
+                              
+                              modal.addEventListener('click',function(){
+                                 modal.style.display = ""none"";
+                                 modalimg.src = """";
+                                 modalimg.alt = """";
+                               })
+                              
+                              thumbnail.addEventListener('click',function(){
+                                 modal.style.display = ""block"";
+                                 modalimg.src = thumbnail.src;
+                                 modalimg.alt = thumbnail.alt;
+                               })";
+        }
+
+        private static string GetDataImageString(ISearchContext context)
+        {
+            ITakesScreenshot newScreen = (ITakesScreenshot)context;
+            return $"{Convert.ToBase64String(newScreen.GetScreenshot().AsByteArray)}";
+        }
+
+        private static string GetCss()
+        {
+            return @"
+                .thumbnail{border: 1px solid black;margin-left:1em;max-width:300px;}
+                .thumbnail:hover{border:2px solid black;}
+                .wrap .wrapTwo .wrapThree{margin:2px;max-width:70vw;}
+                .wrapOne {margin-left:1em;overflow-wrap:anywhere;}
+                .wrapTwo {margin-left:2em;overflow-wrap:anywhere;}
+                .wrapThree {margin-left:3em;overflow-wrap:anywhere;}
+                .emOne {margin-left:1em;overflow-wrap:anywhere;}
+                .emTwo {margin-left:2em;overflow-wrap:anywhere;}
+                .emThree {margin-left:3em;overflow-wrap:anywhere;}
+                #modal {display: none;position: fixed;z-index: 1;padding-top: 100px;left: 0;top: 0;width: 100%;
+                 height: 100%;overflow: auto;background-color: rgba(0, 0, 0, 0.9);}
+                #modalimage {margin: auto;display: block;width: 80%;}
+                .htmlTable{border-top:double lightgray;width:100%;display:table;}
+                .sectionbutton{background-color: #000000; color: #FFFFFF; cursor: pointer; padding: 18px; width: 100%;
+                 text-align: left; outline: none; transition: 0.4s; border: 1px solid black;}
+                .sectionbutton:hover {background-color: #828282;}
+                .buttonInfoText {width: 50%; float: left;}
+                .buttonExpandoText {text-align: right; width: 50%; float: right;}
+                .majorSection{padding: 0 18px;background-color:white;max-height: 0;overflow:hidden;
```

Comment:
Would prefer results weren't automatically hidden - Can we may the default visible?

Reply:
The reason I've hidden them by default is otherwise the collapsible headers/buttons will be offscreen and it won't be immediately obvious that they are there. although I do agree that not having any of the body of the report shown by default is not ideal. I do have another idea on how to display the content in a nice way so I'll make the default visible and do another pass at it later in the week.